### ISSUE TITLE
Unmet peer dependency: eslint-plugin-react

### ISSUE THREAD

**Issue by kinday** [iamstarkov/generator-zen#34]
Got this message during installation:
```
npm WARN deprecated native-or-bluebird@1.2.0: please use 'any-promise' instead
```

**Comment by iamstarkov** [comment]
do you how to find packages which are using `native-or-bluebird`? seems no direct dependency is using this package

**Comment by kinday** [comment]
Oh, sorry. I should’ve checked it first.

I can’t reproduce it now, but have different warning and it’s more relevant to your module deps:
```
npm WARN eslint-config-airbnb@6.1.0 requires a peer of eslint-plugin-react@^4.0.0 but none was installed.
```

**Comment by iamstarkov** [comment]
okay, let's clarify, are you talking about `generator-zen` itself or scaffolded packages?

**Comment by iamstarkov** [comment]
here we go https://github.com/airbnb/javascript/issues/751

**Comment by ljharb** [comment]
See https://github.com/airbnb/javascript/pull/829

**Comment by iamstarkov** [comment]
@ljharb thats awesome, thank you

**Comment by iamstarkov** [comment]
closed in [`v2.1.9`](https://github.com/iamstarkov/generator-zen/releases/tag/v2.1.9)
